src/app/components/home/home.component.spec.ts

```diff
@@ -435,12 +435,14 @@ describe('HomeComponent', () => {
     });
 
     it('should skip creating a chart if one already exists', () => {
+      spyOn(component, 'updateData').and.callFake(function(){});
       component.chart = { update: () => {}, data: { datasets: [] } } as any;
       spyOn(component, 'getChart');
       component.organizationUnits = [organizationUnit];
       component.streams = [stream];
       component.types = [supportedType];
       component.organizationUnitCtrl.setValue(organizationUnit.Unit.Name);
+      component.organizationUnitCtrl.setValue(organizationUnit.Unit.Id);
```

Comment:
Is this needed?

Reply:
Those lines aren't actually doing anything and they are triggering the queryStreams functions, which fails because we are not actually able to query SDS. For this reason, I have removed those lines.